apps/explorer/src/modules/txs/pages/PageTxs.vue

```diff
@@ -28,46 +28,61 @@ const MAX_ITEMS = 50
 })
 export default class PageTxs extends Vue {
   txs: Tx[] = []
-  total = 0
-  loading = true
-  error = false
+  totalTx = 0
+  isLoading = true
+  error = ''
+
+  /*
+  ===================================================================================
+    Lifecycle
+  ===================================================================================
+  */
 
-  // Lifecycle
   mounted() {
     this.fetchTotalTxs().then(
       res => {
-        this.total = res
+        this.totalTx = res
       },
       err => {
-        this.total = 0
+        this.totalTx = 0
       }
     )
     this.getPage(0)
   }
 
-  // Methods
+  /*
+  ===================================================================================
+    Methods
+  ===================================================================================
+  */
+
   fetchTxs(page: number): Promise<Tx[]> {
-    return this.$api.getTxs(this.max, page)
+    return this.$api.getTxs(this.maxItems, page)
   }
 
   fetchTotalTxs(): Promise<number> {
     return this.$api.getTotalNumberOfTxs()
   }
 
-  getPage(_page: number): void {
-    this.loading = true
-    this.fetchTxs(_page).then(
+  getPage(page: number): void {
+    this.isLoading = true
+    this.fetchTxs(page).then(
       res => {
-        this.loading = false
+        this.isLoading = false
         this.txs = res
       },
       err => {
-        this.error = true
+        this.error = `${JSON.stringify(err)}`
```

Comment:
Taking a look on this I prefer to put a generic error message (with API 2 we're going to map properly the errors) but for now let's use the generic message we currently have in the I18N strings list